addon/templates/building/edit/form.hbs

```diff
@@ -41,17 +41,30 @@
         {{t
           "ember-gwr.building.sections.kindOfWorkForProject"
           projectId=@model.projectId
-          ARBID=this.buildingWork.ARBID
         }}
       </h3>
-      <Field
-        @attr="kindOfWork"
-        @type="select"
-        @convertValueTo="number"
-        @gwrEnumOptions={{this.Models.BuildingWork.kindOfWorkOptions}}
-        @required={{true}}
-        @translationBase="ember-gwr.buildingWork"
-      />
+      {{#unless this.buildingWork.isNew}}
+        <div class="uk-flex uk-margin">
+          <div class="uk-width-1-2 uk-paddin-small">
+            <span class="uk-text-bold">{{t
+                "ember-gwr.buildingWork.kindOfWork"
+              }}:</span>
+            <span>{{t
+                (concat
+                  "ember-gwr.buildingWork.kindOfWorkOptions."
+                  this.buildingWork.kindOfWork
+                )
+              }}</span>
+          </div>
+          <div class="uk-width-1-2 uk-paddin-small">
+            <span class="uk-text-bold">{{t
+                "ember-gwr.buildingWork.ARBID"
+              }}:</span>
+            <span>{{this.buildingWork.ARBID}}</span>
+          </div>
+        </div>
+      {{/unless}}
```

Comment:
I would always show the (default) kind of work, so the user knows what they are creating.
```
      <h3>
        {{t
          "ember-gwr.building.sections.kindOfWorkForProject"
          projectId=@model.projectId
        }}
      </h3>
      <div class="uk-flex uk-margin">
        <div class="uk-width-1-2 ">
          <span class="uk-form-label">{{t
              "ember-gwr.buildingWork.kindOfWork"
            }}:</span>
          <span>{{t
              (concat
                "ember-gwr.buildingWork.kindOfWorkOptions."
                this.buildingWork.kindOfWork
              )
            }}</span>
        </div>
        {{#unless this.buildingWork.isNew}}
          <div class="uk-width-1-2 ">
            <span class="uk-form-label">{{t
                "ember-gwr.buildingWork.ARBID"
              }}:</span>
            <span>{{this.buildingWork.ARBID}}</span>
          </div>
        {{/unless}}
      </div>
```

Reply:
Done: removed the label